.circleci/osx_install_dependencies.sh

```diff
@@ -50,7 +50,8 @@ if [ ! -f /usr/local/lib/libz3.a ] # if this file does not exists (cache was not
 then
   brew update
   brew upgrade
-  brew install cmake
+  curl https://raw.githubusercontent.com/Homebrew/homebrew-core/b4e46db74e74a8c1650b38b1da222284ce1ec5ce/Formula/c/cmake.rb >> cmake.rb
```

Comment:
Can you add a comment here about the reason for the downgrading as you have in the PR description? So we can remember to bump it later.

Reply:
done